### DISCUSSION TITLE
"CppEthernet" - How to get started with new eclipse workspace!?

### DISCUSSION THREAD

**Discussion by krlorik** [RobertK66/obc_1769_core#76] [Announcements]
Downloading the lpc1769 in the Procedure it is written to create a new folder with CppEthernet, but it seems so that a folder with that name exist already in the project. Screenshot will follow as better description of the issue.

**Comment by RobertK66** [comment]
CppEthernet !?
I think you refer to another repos here!? ( https://github.com/RobertK66/Climb-Obc-Sandbox ). I referred this as reference to 'what  happend earlier', but I did not visit it for some time now.

Which 'Procedure' do you refer here? Some How To from that repos?

**Comment by RobertK66** [comment]
For using this actual repos here you should prepare a new empty workspace where after git checkout you should get the 2 projects named 'ado_chip_175x_6x' and 'ClimbObc'.

**Comment by krlorik** [comment]
https://github.com/carstenscharlemann/Climb-Obc-Sandbox/blob/master/mddocs/HowTos/AddNewProject.md
![CppEthernet](https://user-images.githubusercontent.com/88948328/221407289-37a6f70d-e997-497f-9260-15644355bc8b.PNG)
this folder already exist by importing the project!
